### ISSUE TITLE
Full localization

### ISSUE THREAD

**Issue by glixx** [bcrowell/opengrade#5]
At the moment translation is partial.
Moving all strings into MyWords.pm will do full localization.

**Comment by bcrowell** [comment]
I agree this is something that should be done, but it's not something that I want to spend time doing. I would be grateful to anyone who wanted to take care of this task, and would be happy to merge a patch from them.
